doc/cluster-management/upgrade.md

```diff
@@ -567,3 +567,144 @@ ocne-control-plane-p55sl   Ready    control-plane   10m     v1.30.3+2.el8
 ocne-md-0-9xzrv-dphhk      Ready    <none>          4m43s   v1.30.3+2.el8
 ocne-md-0-9xzrv-qsbbj      Ready    <none>          2m43s   v1.30.3+2.el8
 ```
+
+### Upgrading OLVM Cluster API Clusters
+
+#### Prepare for the Upgrade
+Follow the instructions to [create](olvm.md/#creating-the-olvm-ock-image) and [upload](olvm.md/#uploading-the-olvm-ock-image) an OLVM OCK image for the Kubernetes version being upgraded to.
+[Create](olvm.md/#creating-a-vm-template) an OLVM VM template to use the OLVM OCK image for the upgrade.
+
+#### Create a Cluster
+
+Instantiate a small cluster with the OLVM provider using Kubernetes 1.30.
```

Comment:
what is small in this case?
why do we need this cluster?

Reply:
I copied the text for upgrading an OCI CAPI cluster.  I will remove the word "small" in both places.  We need this cluster to manage the CAPI cluster.  As noted in the next sentence, this cluster may already exist.

Reply:
I removed the word `small` in three places from this document.